core/src/main/java/io/smallrye/openapi/api/OpenApiConfigImpl.java

```diff
@@ -405,6 +406,17 @@ public OperationIdStrategy getOperationIdStrategy() {
         return null;
     }
 
+    @Override
+    public DuplicateOperationIdBehavior getDuplicateOperationIdBehavior() {
+        if (duplicateOperationIdBehavior == null) {
+            String behavior = getStringConfigValue(OpenApiConstants.DUPLICATE_OPERATION_ID_BEHAVIOR);
+            if (behavior != null) {
+                return DuplicateOperationIdBehavior.valueOf(behavior);
+            }
+        }
+        return null;
```

Comment:
This approach will cache the value and also defer to the interface's default if not found in the configuration properties.

```suggestion
        if (duplicateOperationIdBehavior == null) {
            duplicateOperationIdBehavior = getConfig()
                    .getOptionalValue(OpenApiConstants.DUPLICATE_OPERATION_ID_BEHAVIOR, String.class)
                    .map(DuplicateOperationIdBehavior::valueOf)
                    .orElseGet(OpenApiConfig.super::getDuplicateOperationIdBehavior);
        }
        return duplicateOperationIdBehavior;
```

Reply:
Done.